translate/expression/py_expression.py

```diff
@@ -18,7 +18,8 @@ def expr(node):
 """
 def call(node):
     gast = {}
-    gast["type"] = pr.node_to_gast(node.func)
+    gast["type"] = "func" #pr.node_to_gast(node.func)
```

Comment:
I would be more specific than func for the type to distinguish between a function definition and a function call.
Maybe funcCall?

Reply:
Sounds good